server/services/instrumenter.js

```diff
@@ -1,11 +1,13 @@
 import {_} from 'meteor/underscore';
 import Log from './../context/log';
-import Conf from './../context/conf'
-let hookLoader, instrumentJs, checkIfAutorised, shallInstrumentClientScript, shallInstrumentServerScript
+import Conf from './../context/conf';
+
+let hookLoader, instrumentJs, shouldIgnore, shallInstrumentClientScript, shallInstrumentServerScript;
 
 if (Conf.IS_COVERAGE_ACTIVE) {
     //var im = Npm.require('istanbul-middleware');
     var istanbulAPI = Npm.require('istanbul-api'),
+        minimatch = Npm.require('minimatch'),
```

Comment:
These npm.require can be transformed into import minimatch from "minimatch" right ?